common/src/main/webapp/sass/explore-primereact.scss

```diff
@@ -0,0 +1,164 @@
+body {
```

Comment:
We don't need to enclose the theme imports in `body`. It will save us some space since `body` is copied to every selector in the theme.